src/CountingWords.ts

```diff
@@ -1,13 +1,19 @@
 const countingWords = (phrase: string) => {
   let count = 0
 
-  const wordsArray = phrase.split(' ')
+  try {
+    const wordsArray = phrase.split(' ').filter(word => word != '')
+    
+    if (wordsArray != []) {
+      for (const index in wordsArray) {
+        count++
+      }
+    }
+    return count
 
-  for (const index in wordsArray) {
-    count++
+  } catch (error) {
+    throw new Error(`Error: ${error}`)
   }
-  
-  return count
 }
 
-console.log(countingWords("This is a test of a phrase"))
\ No newline at end of file
+console.log(countingWords(""))
```

Comment:
Para evitar estos errores siempre es conveniente meter una línea en blanco al final de los ficheros (editorconfig te lo hace automáticamente).

![CleanShot 2021-11-10 at 19 36 11@2x](https://user-images.githubusercontent.com/2389019/141181383-f0bed951-5cbb-44ab-8728-a202c05898fe.png)


```suggestion
console.log(countingWords(""))

```